misc/testerina/modules/testerina-runtime/src/main/java/org/ballerinalang/test/runtime/entity/CoverageReport.java

```diff
@@ -187,4 +224,57 @@ private void createReport(final IBundleCoverage bundleCoverage, ModuleCoverage m
         return filteredPathList;
     }
 
+    private void writeReport(final IBundleCoverage bundle) {
+
+        try {
+            final IReportVisitor visitor = createReportVisitor();
+
+            // Not sure what this does. Enable and test
```

Comment:
Lets remove the dev comment. It was something i added when i was testing the feature out. its not required,

Reply:
Looks like there are no usages to this. If so, please remove this.